Jenkinsfile

```diff
@@ -165,9 +200,14 @@ node('vetsgov-general-purpose') {
       for (int i=0; i<envNames.size(); i++) {
         def envName = envNames.get(i)
 
+        def buildCommand = "npm run build -- --content-deployment --buildtype=${envName}"
+        if (isCMSDeploy) {
+          buildCommand = "npm run build -- --content-deployment --brand-consolidation-enabled --buildtype=${envName}"
+        }
```

Comment:
I think the `--content-deployment`  flag should be omitted by default, and only applied when `isCMSDeploy` is true. Maybe something like -

```groovy
def cmsDeployArg = ""
if (isCMSDeploy) {
    cmsDeployArg = "--content-deployment"
}
def buildCommand = "npm run build -- ${cmsDeployArg} --buildtype=${envName}"
```